dockernet/config/evmos_ica.json

```diff
@@ -0,0 +1,24 @@
+{
+    "interchainaccounts": {
+        "host_genesis_state": {
+            "active_channels": [],
+            "interchain_accounts": [],
+            "port": "icahost",
+            "params": {
+                "host_enabled": true,
+                "allow_messages": [
+                    "/cosmos.bank.v1beta1.MsgSend",
+                    "/cosmos.bank.v1beta1.MsgMultiSend",
+                    "/cosmos.staking.v1beta1.MsgDelegate",
+                    "/cosmos.staking.v1beta1.MsgUndelegate",
+                    "/cosmos.staking.v1beta1.MsgBeginRedelegate",
+                    "/cosmos.staking.v1beta1.MsgRedeemTokensforShares",
+                    "/cosmos.staking.v1beta1.MsgTokenizeShares",
```

Comment:
not supported atm
```suggestion
```

Reply:
thanks, updated to reflect https://github.com/evmos/evmos/blob/main/app/upgrades/v11/upgrades.go#L77